NuRadioReco/framework/trigger.py

```diff
@@ -394,9 +395,8 @@ def __init__(self, name, passband, order, threshold, number_of_coincidences=2,
         self._coinc_window = channel_coincidence_window
 
 class RNOGSurfaceTrigger(Trigger):
-    from NuRadioReco.utilities import units
     def __init__(self, name, threshold, number_of_coincidences=1,
-                 channel_coincidence_window=60*units.ns, channels=[13, 16, 19], temperature=250*units.kelvin, Vbias=2*units.volt):
+                 channel_coincidence_window=60, channels=[13, 16, 19], temperature=250, Vbias=2):
```

Comment:
Why does this no longer have units?

Reply:
Since no trigger here had units I thought I continue doing this. But I agree, with units is better!